src/commons/repl/Repl.tsx

```diff
@@ -40,7 +40,12 @@ type OwnProps = {
 
 const Repl = React.forwardRef<AceEditor, ReplProps>((props, ref) => {
   const cards = props.output.map((slice, index) => (
-    <Output output={slice} key={index} usingSubst={props.usingSubst || false} />
+    <Output
+      output={slice}
+      key={index}
+      usingSubst={props.usingSubst || false}
```

Comment:
```suggestion
      usingSubst={props.usingSubst ?? false}
```

Given that the type of `props.usingSubst` is `boolean | undefined`, the nullish coalescing operator better captures the semantics. I understand that you didn't write this code, but it would be good if you could make this change.